client/src/unifycr.c

```diff
@@ -2218,7 +2236,7 @@ static int unifycr_sync_to_del()
     memcpy(cmd_buf + 2 * sizeof(int),
            &local_rank_idx, sizeof(int));
     memcpy(cmd_buf + 3 * sizeof(int),
-           &dbg_rank, sizeof(int)); /*add debug info*/
+           &dbg_rank, sizeof(int)); /*add DEBUG info*/
```

Comment:
I suspect this was an inadvertent search and replace hit (not that it hurts anything :)